src/styles/header.scss

```diff
@@ -0,0 +1,21 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: $white;
+  width: calc(100% - #{$header-padding * 2});
```

Comment:
The variable `$header-padding` is used here but not defined in this file. Ensure it is defined in an imported file or within this SCSS file.